src/eei.cpp

```diff
@@ -440,11 +450,19 @@ Literal EthereumInterface::callImport(Import *import, LiteralList& arguments) {
    * Utilities
    */
 
-  /* Checks if host supplied 256 bit value exceeds UINT128_MAX */
   unsigned int EthereumInterface::exceedsUint128(evm_uint256be const& value)
   {
-    for (unsigned i = 0; i < 16; ++i) {
-      if (value.bytes[i])
+    for (unsigned i = 0; i < 16; i++) {
```

Comment:
You can implement it as:
```
return std::any_of(&value.bytes[0], &value.bytes[16], [](uint8_t b) { return b != 0; });
```
But not forcing it...

Reply:
Can you open a new PR with that change?